packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -3,27 +3,143 @@
+}
+
+async function updateActiveNavigationLink( { url, label } ) {
+	if ( url ) {
+		await page.type( 'input[placeholder="Search or type url"]', url );
+		// Wait for the autocomplete suggestion item to appear.
+		await page.waitForXPath( `//span[@class="block-editor-link-control__search-item-title"]/mark[text()="${ url }"]` );
+		await page.keyboard.press( 'Enter' );
+	}
+
+	if ( label ) {
+		await page.click( '.wp-block-navigation-link__content.is-selected' );
+
+		// Ideally this would be `await pressKeyWithModifier( 'primary', 'a' )`
+		// to select all text like other tests do.
+		// Unfortunately, these tests don't seem to pass on Travis CI when
+		// using that approach, while using `Home` and `End` they do pass.
+		await page.keyboard.press( 'Home' );
+		await pressKeyWithModifier( 'shift', 'End' );
+		await page.keyboard.press( 'Backspace' );
+		await page.keyboard.type( label );
+	}
+}
+
+describe( 'Navigation', () => {
 	beforeEach( async () => {
 		await createNewPost();
 	} );
 
-	it( 'Should add a link with one click', async () => {
+	it( 'allows a navigation menu to be created using existing pages', async () => {
+		// Mock the response from the Pages endpoint. This is done so that the pages returned are always
+		// consistent and to test the feature more rigorously than the single default sample page.
+		await mockPagesResponse( [
+			{
+				title: 'Home',
+				slug: 'home',
+			},
+			{
+				title: 'About',
+				slug: 'about',
+			},
+			{
+				title: 'Contact Us',
+				slug: 'contact',
+			},
+		] );
+
+		// Add the navigation block.
+		await insertBlock( 'Navigation' );
+
+		// Create an empty nav block. The 'create' button is disabled until pages are loaded,
+		// so we must wait for it to become not-disabled.
+		await page.waitForXPath( '//button[text()="Create from all top pages"][not(@disabled)]' );
+		const [ createFromExistingButton ] = await page.$x( '//button[text()="Create from all top pages"][not(@disabled)]' );
+		await createFromExistingButton.click();
+
+		// Snapshot should contain the mocked pages.
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	it( 'allows a navigation menu to be created from an empty menu using a mixture of internal and external links', async () => {
+		// Add the navigation block.
 		await insertBlock( 'Navigation' );
+
+		// Create an empty nav block.
+		await page.waitForSelector( '.wp-block-navigation-placeholder' );
 		const [ createEmptyButton ] = await page.$x( '//button[text()="Create empty"]' );
 		await createEmptyButton.click();
-		await page.waitForSelector( 'input[placeholder="Search or type url"]' );
-		await page.type( 'input[placeholder="Search or type url"]', 'http://example.com' );
 
-		await page.keyboard.press( 'Enter' );
+		// Add a link to the default Navigation Link block.
+		await updateActiveNavigationLink( { url: 'https://wordpress.org', label: 'WP' } );
+
+		// Move the mouse to reveal the block movers. Without this the test seems to fail.
```

Comment:
That's weird, the element is definitely visible on the page and it's focusable too, wonder why `page.click` doesn't work 🤔

Reply:
Yep, there were a few curiosities with these tests 😄 

It looks like there's a chance the movers will be moved to the toolbar (#18685), so hopefully this won't be a problem for too much longer!